.github/workflows/build-container.yml

```
@@ -13,20 +13,30 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - base-image: alpine
+          - machine: ubuntu-20.04
+            base-image: alpine
             os-type: linux-musl
-    runs-on: ubuntu-20.04
+            architecture: x64
+          - machine: actuated-arm64
```

Is there any list of what Actuated machines are available? Any docs that would help us in troubleshooting?

---

I think maintainers should see it? Actions -> (left side label: management) runners

---

I see only regular GitHub runners.

FYI related discussion: https://github.com/open-telemetry/opentelemetry-go/pull/4994#discussion_r1508641578

---

Yes that would have been next issue after there was more information, which runners are even available.
I was told also that they are going to remove the default label somewhere in the near? future.

---

Look like it would be removed rather sooner than later. I suggest changing this line to 

```suggestion
          - machine: actuated-arm64-4cpu-8gb
```

https://github.com/open-telemetry/opentelemetry-go/pull/4994#discussion_r1509178904

---

will update within this scope then 👍

---

https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3277/commits/2cbc49253876099d927e59f27c1ba9ef194c6f53